Agent/FeatureFlags/NRMAFlags.m

```diff
@@ -47,7 +47,8 @@ + (NRMAFeatureFlags) featureFlags
                               NRFeatureFlag_NetworkRequestEvents | 
                               NRFeatureFlag_RequestErrorEvents |
                               NRFeatureFlag_DistributedTracing |
-                              NRFeatureFlag_AppStartMetrics;
+                              NRFeatureFlag_AppStartMetrics |
+                              NRFeatureFlag_AutoCollectLogs;
```

Comment:
`NRFeatureFlag_AutoCollectLogs` must be disabled by default. As my resolved comment indicated. Thanks!